go-controller/pkg/util/util.go

```diff
@@ -188,3 +188,30 @@ func HashForOVN(s string) string {
 	return fmt.Sprintf("a%s", hashString)
 
 }
+
+// UpdateIPsSlice will search for values of v in the slice and update it with n values of same IP family
+func UpdateIPsSlice(s, oldIPs, newIPs []string) []string {
```

Comment:
I can't get from the description what is `v` and `n` :)

It looks that is easy to go wrong here and there can be corner cases like passing empty slices that can break
if you don't have time to add an unit test just add a TODO  to add it, is a good first  issue for someone new to the project ...